src/pattern-library/components/patterns/input/CheckboxPage.js

```diff
@@ -0,0 +1,194 @@
+    <Library.Page
+      title="Checkbox"
+      intro={
+        <p>
+          <code>Checkbox</code> is a composite component that includes a
+          checkbox input and label.
+        </p>
+      }
+    >
+      <Library.Pattern title="Status">
+        <p>
+          <code>Checkbox</code> combines the functionality of both the{' '}
+          <code>Checkbox</code> and <code>LabeledCheckbox</code> legacy
+          components.
+        </p>
+
+        <Library.Example title="Migrating to this component (from Checkbox)">
+          <Next.Changelog>
+            <Next.ChangelogItem status="breaking">
+              Prop name:{' '}
+              <s>
+                <code>onToggle</code>
+              </s>{' '}
+              ➜ Use <code>onChange</code> instead
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="breaking">
+              Prop:{' '}
+              <s>
+                <code>inputRef</code>
+              </s>{' '}
+              ➜ <code>elementRef</code>
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="breaking">
+              The presence of prop <code>checked</code> now configures{' '}
+              <code>Checkbox</code> as a controlled component.
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="added">
+              Prop <code>defaultChecked</code>
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="added">
+              Now accepts HTML attributes allowed for <code>HTMLInput</code>{' '}
+              elements.
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="changed">
+              Prop <code>id</code> is no longer required
+            </Next.ChangelogItem>
+          </Next.Changelog>
+        </Library.Example>
+
+        <Library.Example title="Migrating to this component (from LabeledCheckbox)">
+          <Next.Changelog>
+            <Next.ChangelogItem status="breaking">
+              Prop:{' '}
+              <s>
+                <code>containerClasses</code>
+              </s>{' '}
+              ➜ this is now a composite component (does not take extra CSS{' '}
+              <code>classes</code>)
+            </Next.ChangelogItem>
+          </Next.Changelog>
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Usage">
+        <Next.Usage componentName="Checkbox" />
+        <Library.Example>
+          <Library.Demo title="Basic Checkbox" withSource>
+            <Checkbox>Click me</Checkbox>
+          </Library.Demo>
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Props">
+        <Library.Example title="checked">
+          <p>
+            When the <code>checked</code> prop is present, <code>Checkbox</code>{' '}
+            will behave as a controlled component.
+          </p>
+          <Next.Code
+            content={`const [controlledChecked, setControlledChecked] = useState(false);
```

Comment:
I'd suggest removing the word "controlled" from the identifiers here, to make the code a bit less verbose.